### PR TITLE
feat(apisix-dashboard): add etcd prefix

### PR DESCRIPTION
**Pull request by Gallardot**
[Refer to Dashboard Configuration](https://github.com/apache/apisix-dashboard/blob/1fa139f5316805f6f2ae765434ab47615256bc3d/api/conf/conf.yaml#L48)

### CODE REVIEW

**Comment by tao12345666333** [comment]
Thanks!

File: charts/apisix-dashboard/templates/configmap.yaml [hunk 1]
```diff
@@ -30,6 +30,7 @@ data:
         port: {{ .listen.port }}
       {{- with .etcd }}
       etcd:
+        prefix: {{ .prefix | quote }}
```
Comment (tokers) [inline charts/apisix-dashboard/templates/configmap.yaml:33]: Could you also describe it in the README?

Comment (Gallardot) [reply charts/apisix-dashboard/templates/configmap.yaml:33]: > Could you also describe it in the README?

@tokers @juzhiyuan 

Maybe you mean something like this [configuration list description](https://github.com/apache/apisix-helm-chart/tree/master/charts/apisix#parameters)?  Maybe we can create another PR to handle this.

Comment (tokers) [reply charts/apisix-dashboard/templates/configmap.yaml:33]: Yes, but I think it's quite handful to add some descriptions for it, maybe in the same PR is better.

Comment (Gallardot) [reply charts/apisix-dashboard/templates/configmap.yaml:33]: Done! 

@juzhiyuan @tokers  PTAL!

**Review by nic-chen — APPROVED** [review]
LGTM

File: charts/apisix-dashboard/templates/configmap.yaml (hunk 1 above)
Comment (tokers) [reply charts/apisix-dashboard/templates/configmap.yaml:33]: @Gallardot Thanks!

**Review by tokers — APPROVED** [review]

**Pull request merged**